packages/manager/core/api/src/filters.ts

```diff
@@ -67,10 +68,24 @@ export function applyFilters<T>(items: T[] = [], filters: Filter[] = []) {
           keep = keep && `${value}`.toLowerCase().endsWith(comp.toLowerCase());
           break;
         case FilterComparator.IsEqual:
-          keep = keep && `${value}`.toLowerCase() === comp.toLowerCase();
+          if (filter.type === FilterTypeCategories.Date) {
+            keep =
+              keep &&
+              new Date(`${value}`).setHours(0, 0, 0, 0) ===
+                new Date(comp).setHours(0, 0, 0, 0);
```

Comment:
As your example also displays time, i think this can contradict the time part and maybe we need to accept time as well in the filtering.

Reply:
That's a good point. I assumed filtering on dates would only be done on dates without time.
I'll ask QA for more information about this point

Reply:
@anooparveti I just had confirmation that it's only date filtering (ISO with prod)